src/Scenes/CheckLocation.fs

```diff
@@ -20,7 +20,7 @@ type Status =
 | Changed
 | Error of string
 
-type Model =
+type [<Pojo>] Model =
```

Comment:
I never had to add this!? I didn't even know it existed. It's not in the suave fable sample?!